packages/senchabot-prisma/prisma/seed.ts

```diff
@@ -18,47 +67,197 @@ main()
     process.exit(1);
   });
 
-async function addCommandsAndAliases(twitchChannelId: string) {
+const discordDataCreate = async (discordUserId: string | null) => {
+  if (discordUserId) {
+    await prisma.discordServer.create({
```

Comment:
If we do it this way, I think we can get the serverId value from `discordServer`.
```suggestion
    const discordServer = await prisma.discordServer.create({
```